### DISCUSSION TITLE
The webui does not show NETWORK section on version 4.2.1

### DISCUSSION THREAD

**Discussion by janusn** [nicolargo/glances#3026] [Q&A] [answered]
The webui does not show NETWORK section illustrated as following:
![Screenshot 2024-11-20 at 23 30 14](https://github.com/user-attachments/assets/fcaaa4a0-20f7-4d17-8a58-07a69d2e75e5)

But the NETWORK section is show on terminal illustrated as following:
![Screenshot 2024-11-20 at 23 30 32](https://github.com/user-attachments/assets/b94338fa-5fa6-41dc-8f61-a6bad07cfbec)

Both are run with the same config file and parameters except -w.
The NETWORK section was shown on the webui with version 4.2.0.

Any pointer? Thanks.

**Comment by Villantiqua** [comment, marked as answer]
I had the same. Put show=enp1s0 in the Network section of Glances.conf replacing the network interface name with yours, e.g., wlan0 or eth0 . Did it for me 🙂

**Reply by janusn** [reply]
Thanks for the info. I did tried `show=en*` but it didn't work.

It has to be precise. 😅

**Comment by Villantiqua** [comment]
Sure, you’ve got to use the exact name of your network interface.On 23 Nov 2024, at 17:12, janusn ***@***.***> wrote:﻿
Thanks for the info. I did tried show=en* but it didn't work.
It has to be precise. 😅
